app/src/main/java/org/dhis2/usescases/settings/SyncManagerFragment.java

```diff
@@ -115,6 +124,7 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
     @Override
     public void onResume() {
         super.onResume();
+        context.registerReceiver(networkReceiver, new IntentFilter("android.net.conn.CONNECTIVITY_CHANGE"));
```

Comment:
I think this was deprecated in 28 
https://developer.android.com/reference/android/net/ConnectivityManager#CONNECTIVITY_ACTION

Reply:
the value of the constant action is actually Constant Value: "android.net.conn.CONNECTIVITY_CHANGE"

Reply:
In fact it is the registration of the receiver in the manifest what is deprecated. It is valid to use if it is registered programmatically using a valid context.